src/CoverageTransformer.js

```diff
@@ -0,0 +1,246 @@
+const { Collector } = require('istanbul');
+const path = require('path');
+const fs = require('fs');
+const { SourceMapConsumer } = require('source-map');
+
+const sourceMapRegEx = /(?:\/{2}[#@]{1,2}|\/\*)\s+sourceMappingURL\s*=\s*(data:(?:[^;]+;)+base64,)?(\S+)/;
+
+const SparceCoverageCollector = require('./SparceCoverageCollector').SparceCoverageCollector;
+
+const getMapping = require('./getMapping');
+
+const remapFunction = require('./remapFunction');
+const remapBranch = require('./remapBranch');
+
+
+class CoverageTransformer {
+  constructor(options) {
```

Comment:
earlier comment about tabs vs. spaces... won't repeat myself further on this. :)